ios/FluentUI/Vnext/Shimmer/ShimmerModifiers.swift

```diff
@@ -5,6 +5,26 @@
 
 import SwiftUI
 
+public extension ShimmerView {
+    /// Sets the accessibility label for the Shimmer.
+    /// - Parameter accessibilityLabel: Accessibility label string.
+    /// - Returns: The modified Shimmer with the property set.
+    func accessibilityLabel(_ accessibilityLabel: String?) -> ShimmerView {
+        state.accessibilityLabel = accessibilityLabel
+        return self
+    }
+}
+
+public extension ShimmerLinesView {
+    /// Sets the accessibility label for the ShimmerLines.
+    /// - Parameter accessibilityLabel: Accessibility label string.
+    /// - Returns: The modified ShimmerLines with the property set.
+    func accessibilityLabel(_ accessibilityLabel: String?) -> ShimmerLinesView {
```

Comment:
Why do we need this? Does the default accessibility label modifier from SwiftUI not work?